libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar-inner.component.ts

```diff
@@ -256,23 +259,24 @@ export class SkyDatepickerCalendarInnerComponent
     return false;
   }
 
-  public onKeydown(event: KeyboardEvent) {
-    const key = this.keys[event.which];
+  public onKeydown(event: KeyboardEvent): void {
```

Comment:
Where is this called from now? I am not seeing it anywhere and it is removed in your changes in the HTML.

Reply:
It's call from the parent now: `datepicker-calendar.component.html` line 15